uint32 id
float32[2] top_left
float32[2] bottom_right
float32 area